internal/podtemplatespec/stub.go

```diff
@@ -0,0 +1,18 @@
+package podtemplatespec
+
+import corev1 "k8s.io/api/core/v1"
+
+type Stub struct {
+	pts corev1.PodTemplateSpec
+}
+
+func NewStub(p corev1.PodTemplateSpec) *Stub {
```

Comment:
I don't think this definition is necessary.

Reply:
Removed `stub.go` entirely as it is trivial and leaving it up to the test code in the interface caller to implement their own stubs.